File: pset6/cash.py
from cs50 import get_float
from math import floor

while True:
    dollars_owed=get_float("change owed:\n")
    cents_owed=floor(dollars_owed*100)
    if cents_owed>0.00:
        break

quarters = cents_owed // 25
dimes = (cents_owed % 25) // 10
nickels = ((cents_owed % 25) % 10) // 5
pennies = ((cents_owed % 25) % 10) % 5

print(f"{quarters + dimes + nickels + pennies}")

File: pset1/cash.c
#include<cs50.h>
#include<math.h>
#include<stdio.h>
int main(void)
{
    float b;
do
{
    b=get_float("change owed:\n");
}
while(b<=0.00);
{ 
    int cents= round(b* 100);
{
int coins=0;
while(cents>=25)
{ coins++;
cents=cents-25;
}
while(cents>=10)
{ coins++;
cents=cents-10;
}
while(cents>=5)
{ coins++;
cents=cents-5;
}
while(cents>=1)
{ coins++;
cents=cents-1;
}
{
printf("%i\n",coins);
}
}
}
}
